src/main/java/UI.java

```diff
@@ -41,36 +45,58 @@ public Space getNextMove(Game game) {
         }
     }
```

Comment:
Alright, I know this PR barely touched this `switch` statement, but let's clean up the duplication among these cases. I imagine that some division / modulo arithmetic will solve this more compactly.